vips/vip-324/bsctestnet.ts

```diff
@@ -0,0 +1,250 @@
+    supplySpeedPercentage: 100,
+    borrowSpeedPercentage: 0,
+    distributor: LST_XVS_DISTRIBUTOR,
+  },
+];
+
+export const BRIDGE_FEES = parseUnits("0.4", 18);
+export const DEST_ENDPOINT_ID = 10161; // Sepolia chain
+export const ADAPTER_PARAMS = ethers.utils.solidityPack(["uint16", "uint256"], [1, 300000]);
+export const PERCENTAGE_OF_REAL_AMOUNT_BRIDGE = 10;
+export const TOTAL_MONTHS = 3;
+export const TOTAL_XVS_TO_BRIDGE = CORE_SPEEDS.concat(CURVE_SPEEDS)
+  .concat(LST_SPEEDS)
+  .reduce((acc, { reward }) => acc.add(reward), ethers.BigNumber.from(0))
+  .mul(TOTAL_MONTHS)
+  .div(PERCENTAGE_OF_REAL_AMOUNT_BRIDGE);
+
+export const TOTAL_XVS_FOR_CORE = CORE_SPEEDS.reduce(
+  (acc, { reward }) => acc.add(reward),
+  ethers.BigNumber.from(0),
+).mul(TOTAL_MONTHS);
+export const TOTAL_XVS_FOR_CURVE = CURVE_SPEEDS.reduce(
+  (acc, { reward }) => acc.add(reward),
+  ethers.BigNumber.from(0),
+).mul(TOTAL_MONTHS);
+export const TOTAL_XVS_FOR_LST = LST_SPEEDS.reduce((acc, { reward }) => acc.add(reward), ethers.BigNumber.from(0)).mul(
+  TOTAL_MONTHS,
+);
+
+export const BLOCKS_PER_YEAR = BigNumber.from(2628000); // assuming a block is mined every 12 seconds
+export const BLOCKS_IN_30_DAYS = BLOCKS_PER_YEAR.mul(30).div(365);
+const commands = CORE_SPEEDS.concat(CURVE_SPEEDS)
+  .concat(LST_SPEEDS)
+  .map(speed => {
+    const totalAmount = speed.reward;
+    const supplyAmount = totalAmount.mul(speed.supplySpeedPercentage).div(100);
+    const supplySpeed = supplyAmount.div(BLOCKS_IN_30_DAYS);
+    const borrowAmount = totalAmount.mul(speed.borrowSpeedPercentage).div(100);
+    const borrowSpeed = borrowAmount.div(BLOCKS_IN_30_DAYS);
+
+    return {
+      target: speed.distributor,
+      signature: "setRewardTokenSpeeds(address[],uint256[],uint256[])",
+      params: [[speed.address], [supplySpeed], [borrowSpeed]],
+      value: "0",
+      dstChainId: DEST_ENDPOINT_ID,
+    };
+  });
+
+export const vip324 = () => {
+  const meta = {
+    version: "v2",
+    title: "VIP-322",
+    description: ``,
+    forDescription: "Execute this proposal",
+    againstDescription: "Do not execute this proposal",
+    abstainDescription: "Indifferent to execution",
+  };
+
+  return makeProposal(
+    [
+      {
+        target: XVS,
+        signature: "approve(address,uint256)",
+        params: [XVS_BRIDGE_SRC, TOTAL_XVS_TO_BRIDGE],
+      },
+      {
+        target: XVS_BRIDGE_SRC,
+        signature: "sendFrom(address,uint16,bytes32,uint256,(address,address,bytes))",
+        params: [
+          NORMAL_TIMELOCK,
+          DEST_ENDPOINT_ID,
+          ethers.utils.defaultAbiCoder.encode(["address"], [ETH_TREASURY]),
+          TOTAL_XVS_TO_BRIDGE,
+          [BSC_TREASURY, ethers.constants.AddressZero, ADAPTER_PARAMS],
+        ],
+        value: BRIDGE_FEES.toString(),
+      },
+      {
+        target: XVS,
+        signature: "approve(address,uint256)",
+        params: [XVS_BRIDGE_SRC, 0],
+      },
+      {
```

Comment:
Need a command to accept the ownership of Treasury

Reply:
You can simulate it once before the execution of multisig by adding it to the before block of sepolia simulations of multichain VIP.